components/builder-api/habitat/default.toml

```diff
@@ -1,3 +1,6 @@
+[srv]
+port = 9636
```

Comment:
It's something that should be used to configure the port, but right now we read the `config.toml` that the Supervisor generates directly. We'll need to change it up just a bit soon. I'm going to take a pass on the builder services as soon as the `pkg_bind_required`/`pkg_bind_optional` work is done and this will get picked up in that work